elk/training/eigen_reporter.py

```diff
@@ -146,3 +195,27 @@ def fit(
         assert x_pos.shape == x_neg.shape
         self.update(x_pos, x_neg)
         return self.fit_streaming()
+
+    def platt_scale(
+        self, labels: Tensor, x_pos: Tensor, x_neg: Tensor, max_iter: int = 100
+    ):
+        """Fit the scale and bias terms to data with LBFGS."""
+
+        opt = optim.LBFGS(
+            self.parameters(),
```

Comment:
If we ever add any other learnable parameters to EigenReporter this could go unnoticed and lead to some bugs (not sure how big of an issue this is though). I favor `[self.bias, self.scale]`.

Reply:
Can you commit this